site/en/docs/privacy-sandbox/overview/index.md

```diff
@@ -179,3 +205,25 @@ core principles underlying the APIs.
 * Privacy Sandbox overview: [Building a more private web](https://www.blog.google/products/chrome/building-a-more-private-web/)
 * Google AI Blog: [Federated Learning: Collaborative Machine Learning without Centralized Training Data](https://ai.googleblog.com/2017/04/federated-learning-collaborative.html)
 * [The future of third-party cookies](https://blog.chromium.org/2019/10/developers-get-ready-for-new.html)
+
+### Stay up to date on the progress of the Privacy Sandbox
+
+You can follow the monthly updates to the [Progress in the Privacy Sandbox](/tags/progress-in-the-privacy-sandbox/) series of articles which also includes an [RSS / Atom feed where you can subscribe](/feeds/progress-in-the-privacy-sandbox.xml) with your preferred reader.
+
+The article series links to the matching monthly updates to the [Privacy Sandbox timeline](https://privacysandbox.com/timeline/) which shows the current status and schedule for proposals.
+
+These high-level resources will provide signposts to changes across the project, but for individual proposals where you want to follow in detail you should:
+
+ - Watch or Star proposal repos on GitHub to get notification of new issues and updates: the Privacy Sandbox [status page](/docs/privacy-sandbox/status/) provides a link to the repo for each proposal
+ - Join the associated [W3C group](https://www.w3.org/groups/) for regular meetings discussing the proposal detail
+ - Star the associated entry on [Chrome Platform Status](https://chromestatus.com) for email updates on Chrome implementation changes.
+
+ ### Get involved
```

Comment:
Spacing here may break the markdown!

Reply:
I'm not seeing it in staging but maybe I'm looking in the wrong place. Where would it be broken? thanks!

Reply:
That's just something I've observed previously -- if it's not actually breaking it, it's not a big deal. But the space before the `###` is what I was referring to.

```suggestion
### Get involved
```